### ISSUE TITLE
Create two screens for the demo section

### ISSUE THREAD

**Issue by anjali9265** [zympl/leeway-website#99]
In the demo section of the website, we need two screens as shown in the screenshot below.
Home screen | Transaction screen
-|-
<img width="300" alt="Screenshot 2023-06-18 at 19 10 25" src="https://github.com/zympl/leeway-website/assets/29055976/fde3ed13-4ddf-4794-8b69-ef20bca1a8b5">|<img width="300" alt="Screenshot 2023-06-18 at 19 10 55" src="https://github.com/zympl/leeway-website/assets/29055976/f2d6f633-0fb6-44a3-801f-9dbbbed98225">

To achieve this we need to have two sections side by side similar to the given screenshot.

<img src="https://github.com/zympl/leeway-website/assets/29055976/ea5e47aa-38e3-43c3-9bb1-82a42798c6c5" width="600">

We'll be implementing the home screen on the left section (red background) and the transaction screen on the right(blue background). Please reach out to @ajayesivan or @anjali9265 if you have any queries regarding the issue.

**Comment by Adhithyarajan** [comment]
Do i have to give the colour?

**Comment by ajayesivan** [comment]
@Adhithyarajan For now you can use the green & red backgrounds. We will remove it in a follow-up PR.

Before starting your work on this please finish #108 and get it merged. This is something we have to do on top of the empty shell.
